pkg/daemon/ceph/client/config.go

```diff
@@ -302,3 +302,31 @@ func WriteCephConfig(context *clusterd.Context, clusterInfo *ClusterInfo) error
 	}
 	return nil
 }
+
+// SetConfig applies a setting for a single mgr daemon
+func SetConfig(context *clusterd.Context, clusterInfo *ClusterInfo, daemonID string, key, val string, force bool) (bool, error) {
```

Comment:
This feels redundant with what `MonStore` does in this package. Can we refactor?

Reply:
How about in a separate PR? Agreed that it's redundant. There is just a slightly different behavior between the two and I don't want to block this PR.

Reply:
Sounds good, please open an issue so we don't lose track of it. Thanks!

Reply:
#8107